gradle.properties

```diff
@@ -23,3 +23,4 @@ slf4j_version=1.7.12
 swagger_version=1.3.12
 hive_version=1.2.1
 hadoopcore_version=1.2.1
+hivemetastore_version=1.2.1
```

Comment:
Can we use the hive_version property? This way we use a consistent version of hive modules.